deepchem/models/torch_models/layers.py

```diff
@@ -0,0 +1,36 @@
+import math
+try:
+  import torch
+  import torch.nn as nn
+except:
+  raise ImportError('These classes require Torch to be installed.')
+
+
+class ScaleNorm(nn.Module):
+  """Apply Scale Normalization to input.
+
+    All G values are initialized to sqrt(d).
```

Comment:
Can you add more docs here explaining the class and giving a usage example? In general you want a solid paragraph along with a good usage example

Reply:
I have changed the layer explanation and added a usage example like I did for the MATFeaturizer. Is the updated explanation enough or should I try to add a bit more?

Reply:
You can add an additional line mentioning why / where `ScaleNorm` would be useful?

Reply:
I've added a bit for that now.